app/static/js/manageVolunteers.js

```diff
@@ -49,32 +49,48 @@ $(document).ready(function() {
     });
 
     function updateSelectVolunteer(){
-      $("#addVolunteerModal input[type=checkbox]").each(function(index, checkbox){
-          if(checkbox["checked"] == true){
-            $("#addVolunteersButton").prop("disabled", false)
-            return false
-          }
-          $("#addVolunteersButton").prop("disabled", true)
-      })
+      let selectedCheckboxes = getSelectedCheckboxes()
+      let buttonContent = $("#addVolunteersButton").html()
+      if (selectedCheckboxes.length > 1) {
+        if ($("#addVolunteersButton").html().charAt(buttonContent.length-1) != "s") {
+          // make the button text plural if there are multiple users selected
+          $("#addVolunteersButton").html(buttonContent + "s")
+        }
+      } else if ($("#addVolunteersButton").html().charAt(buttonContent.length-1) == "s") {
+        // remove the s if it is plural and we have less than 2 volunteers
+        $("#addVolunteersButton").html(buttonContent.slice(0, -1))
+      }
+      // disable the submit button if there are no selectedCheckboxes
+      if (selectedCheckboxes.length <= 0) {
+        $("#addVolunteersButton").prop("disabled", true)
+      } else {
+        $("#addVolunteersButton").prop("disabled", false)
+      }
+
     }
     
+    function getSelectedCheckboxes() {
```

Comment:
Perhaps rename this function to getCheckedVolunteers or something to that effect. Doing so would clear up what this function is returning